src/api/status/public/LICENSE

```diff
@@ -0,0 +1,10 @@
+=========================================================
```

Comment:
Not that one, this one: https://github.com/creativetimofficial/material-dashboard/blob/master/LICENSE.md

Reply:
Noted